cli/image.go

```diff
@@ -77,7 +78,7 @@ func (i *ImageCommand) runImages(args []string) error {
 }
 
 // imagesExample shows examples in images command, and is used in auto-generated cli docs.
-// TODO: add example
 func imagesExample() string {
-	return ""
+	return `$ pouch images
+IMAGE ID: bbc3a0323522, IMAGE NAME: docker.io/library/busybox:latest, SIZE: 2.64 KB`
```

Comment:
Does pouch show result this way?
I think maybe it is not. Is it in the following way:
```
$ pouch images ls
IMAGE ID            IMAGE NAME                           SIZE
bbc3a0323522        docker.io/library/busybox:latest     2699
0153c5db97e5        docker.io/library/redis:alpine       2035
```

Reply:
Maybe it caused by containerd 1.0.0. I have a try with the PR #236, here is the result on my environment.
[root@localhost ~]# pouch images ls
Error: unknown command "ls" for "pouch images"
[root@localhost ~]# pouch images
IMAGE ID             IMAGE NAME                                               SIZE
bbc3a0323522         docker.io/library/busybox:latest                         2.64 KB
b81f317384d7         docker.io/library/nginx:latest                           1.98 KB

Reply:
Anyway I think `IMAGE ID: bbc3a0323522, IMAGE NAME: docker.io/library/busybox:latest, SIZE: 2.64 KB` is not correct, because it is only one line.

Reply:
Maybe I should open an issue for #236, I would do that soon and fix this PR. Sorry for that.

Reply:
OK, I will do that immediately

Reply:
That could not better. Maybe I missed something. Thanks a lot.

Reply:
PTAL@allencloud 
I have fix it.